doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc

```diff
@@ -107,6 +107,10 @@ For a complete list of supported dialects, see the _Hibernate SQL Dialects_ tabl
 .. *Host*: Enter the host name of the external database server.
 .. *Port*: Enter the port number of the external database server.
 .. *Jdbc URL*: Enter the JDBC URL for the external database server.
```

Comment:
Important - this needs a preview built so we can see how it renders:

There is no `+` between L109 and L110 and I think this will cause a rendering issue.